### PR TITLE
[added] [classes] Added classes to ProtoStuffSerializer file

### PR DESCRIPTION
**Pull request by anirudh-hegde**
### Purpose of this pull request

Added classes for specific needs.

### Does this PR introduce _any_ user-facing change?
Yes

### CODE REVIEW

**Review by anirudh-hegde — COMMENTED** [review]
I executed mvn spotless:check and mvn spotless:apply
I don't know why checks are taking so long.

File: seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java
Comment (anirudh-hegde) [inline seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java:1]: Please don't merge this as I did not contribute to this file in rim branch.

**Comment by Hisoka-X** [comment]
> I executed mvn spotless:check and mvn spotless:apply I don't know why checks are taking so long.

Maybe it will download some jar in first time.

File: seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java
Comment (Hisoka-X) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java:1]: I believe this is a mistake of your git operation.
image

Comment (anirudh-hegde) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java:1]: > I believe this is a mistake of your git operation. image

I want to delete the file commit https://github.com/apache/seatunnel/pull/6205/commits/843e735074226b0c0abb36633bb003642e3b3f97

Comment (hailin0) [reply seatunnel-common/src/main/java/org/apache/seatunnel/common/constants/JobMode.java:1]: revert this file updates

**Comment by anirudh-hegde** [comment]
image
dead links found!!

**Comment by anirudh-hegde** [comment]
> image dead links found!!

I fixed the dead links

**Comment by anirudh-hegde** [comment]
Hey someone please review my code

**Pull request closed without merging**